test/platform_database.py

```diff
@@ -149,6 +149,26 @@ def test_simplify_verbose_data(self):
         self.assertEqual(self.pdb.get('1337', verbose_data=True), platform_data)
         self.assertEqual(self.pdb.get('1337'), platform_data['platform_name'])
 
+    def test_update_from_web(self):
+        with patch("requests.get") as _get:
+            self.assertEqual(self.pdb.get('1337'), None)
+            _result = MagicMock()
+            _result.configure_mock(**{
+                'json.return_value': [
+                    {
```

Comment:
maybe could be good to document this json format

Reply:
Good point. FYI, this is a test.